docs/src/Modal.doc.js

```diff
@@ -1,17 +1,26 @@
 // @flow strict
 import type { Node } from 'react';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
 import PageHeader from './components/PageHeader.js';
-import Card from './components/Card.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
   <PageHeader
     name="Modal"
-    description="Modals are light temporary views; general purpose containers allowing for a multitude of design choices."
+    description="Modals display content that requires user interaction on a layer above the page content. Modals block the content underneath, preventing users from interacting with anything else besides the Modal.
+
+    The most common example of Modal is confirming an action the user has taken. Modal also provides some of the padding required for styling out of the box."
```

Comment:
Not sure if the second sentence (about padding) makes sense here. This paragraph is an overview and that's more of a detail.